tests/modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ASHLAR } from '../../../../../modules/modules/nf-core/ashlar/main.nf'
+// we zero out the UUID of output tiff images with ZERO_UUID so we get a consistent md5sum
+include { ZERO_UUID } from './zero_uuid.nf'
+
+def TEST_IMG = "/home/pollen/HITS/nextflow/mcmicro/exemplar-001/raw/exemplar-001-cycle-0{6,7}.ome.tiff"
```

Comment:
Absolute paths won't work here. How do other module tests handle paths?

Reply:
I just successfully tested using the new images you posted.  Will update with relative paths when they are merged.